hudi-common/pom.xml

```diff
@@ -286,5 +286,12 @@
       <artifactId>disruptor</artifactId>
       <version>${disruptor.version}</version>
     </dependency>
+
+    <dependency>
```

Comment:
Shouldn't this already be included? Perhaps the build is failing for some profile. Can you share the build command?

Reply:
```
mvn clean package install -DskipTests -Dspark3.3 -Dscala-2.12 -Dhadoop.version=3.3.0 -Dhive.version=3.1.2 -Phudi-platform-service  -Dcheckstyle.skip=true -Drat.skip=true -Pm1-mac -Pthrift-gen-source -Pflink1.14
```
It's due to `-Dhadoop.version=3.3.0`?

Reply:
I just tried it, this problem will occur when using Spark3.3 or Spark3.4.